Favor/Favor/Sources/Scenes/Auth/ViewControllers/SignUpVC.swift

```diff
@@ -0,0 +1,196 @@
+  
+  // MARK: - Properties
+  
+  // MARK: - UI Components
+  
+  private lazy var emailTextField: FavorTextField = {
+    let textField = FavorTextField()
+    textField.delegate = self
+    textField.placeholder = "이메일"
+    textField.updateMessage("실제 사용하는 이메일을 입력해주세요.", for: .info)
+    textField.keyboardType = .emailAddress
+    textField.autocapitalizationType = .none
+    textField.becomeFirstResponder()
+    return textField
+  }()
+  
+  private lazy var pwTextField: FavorTextField = {
+    let textField = FavorTextField()
+    textField.delegate = self
+    textField.placeholder = "비밀번호"
+    textField.updateMessage("영문, 숫자 혼용 8자 이상", for: .info)
+    textField.keyboardType = .asciiCapable
+    textField.autocapitalizationType = .none
+    return textField
+  }()
+  
+  private lazy var pwValidateTextField: FavorTextField = {
+    let textField = FavorTextField()
+    textField.delegate = self
+    textField.placeholder = "비밀번호 확인"
+    textField.keyboardType = .asciiCapable
+    textField.autocapitalizationType = .none
+    return textField
+  }()
+  
+  private lazy var nextButton: LargeFavorButton = {
+    let button = LargeFavorButton(with: .white, title: "다음")
+    button.isEnabled = false
+    return button
+  }()
+  
+  private lazy var vStack: UIStackView = {
+    let stackView = UIStackView()
+    stackView.axis = .vertical
+    stackView.spacing = 72.0
+    [
+      self.emailTextField,
+      self.pwTextField,
+      self.pwValidateTextField,
+      self.nextButton
+    ].forEach {
+      stackView.addArrangedSubview($0)
+    }
+    return stackView
+  }()
+  
+  // MARK: - Life Cycle
+  
+  // MARK: - Binding
+  
+  func bind(reactor: SignUpReactor) {
+    // Action
+    self.emailTextField.rx.text
+      .orEmpty
+      .distinctUntilChanged()
+      .map { Reactor.Action.emailTextFieldUpdate($0) }
+      .bind(to: reactor.action)
+      .disposed(by: self.disposeBag)
+    
+    self.pwTextField.rx.text
+      .orEmpty
+      .distinctUntilChanged()
+      .map { Reactor.Action.passwordTextFieldUpdate($0) }
+      .bind(to: reactor.action)
+      .disposed(by: self.disposeBag)
+    
+    self.pwValidateTextField.rx.text
+      .orEmpty
+      .distinctUntilChanged()
+      .map { Reactor.Action.checkPasswordTextFieldUpdate($0) }
+      .bind(to: reactor.action)
+      .disposed(by: self.disposeBag)
+    
+    self.nextButton.rx.tap
+      .map { Reactor.Action.nextButtonTap }
+      .bind(to: reactor.action)
+      .disposed(by: self.disposeBag)
+    
+    // State
+    reactor.state.asObservable()
+      .map { $0.isEmailValid }
+      .distinctUntilChanged()
+      .skip(1)
+      .subscribe(onNext: { emailValidate in
+        switch emailValidate {
+        case .empty, .invalid:
+          self.emailTextField.updateMessage(emailValidate.description, for: .error)
+        case .valid:
+          self.emailTextField.updateMessage(nil, for: .info)
+        }
+      })
+      .disposed(by: self.disposeBag)
```

Comment:
```suggestion
      .withUnretained(self)
      .subscribe { owner, passwordValidate in
        switch passwordValidate {
        case .empty, .invalid:
          owner.pwTextField.updateMessage(passwordValidate.description, for: .error)
        case .valid:
          owner.pwTextField.updateMessage(nil, for: .info)
        }
      }
      .disposed(by: self.disposeBag)
```

메모리 누수를 방지하기 위해서 
`self`를 참조할 때에는 위와 같은 오퍼레이터를 사용하면 좋을 것 같아!